src/common/exec_utils.h

```diff
@@ -622,6 +622,51 @@ inline nnvm::Graph AssignContext(nnvm::Graph g,
   return g;
 }
 
+inline void CopyGraph(nnvm::Graph *dst, const nnvm::Graph &src, bool copy_variables) {
```

Comment:
since this is used in multiple files, could we not inline it and put it in an implementation file (exec_utils.cc) to reduce code bloat.

Reply:
sure, I will put the functions we introduced in .cc file. However, putting other functions in exec_utils.cc should be done in another PR.